wikimedia/wikipedia-ios
# Add user menu button to Watchlist

**Phabricator:** https://phabricator.wikimedia.org/T335606 (partially)

### Notes
Points to new Components revision that includes the SwiftUI based menu button. Also adds delegate conformance for the button actions and diff tapping originating from the Watchlist view. Component implementation in PR: 

### Test Steps
1. Open the watchlist view
2. Confirm the user menu button are presented and tapping them opens a context menu
